src/rebuild/srv.c

```diff
@@ -1037,14 +1037,62 @@ rebuild_leader_start(struct ds_pool *pool, uint32_t rebuild_ver,
 	return rc;
 }
 
+#if 0
+struct rebuild_stop_arg {
```

Comment:
Did you mean to add this #if 0'd code here?

Reply:
ah, this needs to be removed. sorry.